### ISSUE TITLE
Twitter support

### ISSUE THREAD

**Issue by srid** [erikdesjardins/no-emoji#15]
I can confirm that it doesn't work on Twitter.

On Chromium.

**Comment by mike29107** [comment]
Indeed, please add Twitter support, idiotic emojis are totally out of control on Twitter. Otherwise a great addon. :-P
